pkgs/development/python-modules/beanprice/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, beancount
+, buildPythonPackage
+, click
+, fetchFromGitHub
+, pytestCheckHook
+, python-dateutil
+, regex
+, requests
+, setuptools
+}:
+buildPythonPackage rec {
+  pname = "beanprice";
+  version = "0-unstable-2024-06-19";
```

```suggestion
  version = "1.2.1-unstable-2024-06-19";
```
https://github.com/beancount/beanprice/blob/e894c9182f4d16f9a46ccb87bdaeca1a7dede040/setup.py#L12